### PR TITLE
TVIST1: Update case form types

### PR DESCRIPTION
**Pull request by jekuaitk**
* Makes `bringerPhone` non-required
* Makes `leaseType` non-required
* Renames some property displays
* Changes `leaseRegulatedRent`

### CODE REVIEW

File: migrations/Version20220506111917.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        // If leaseRegulatedRent is null we assume lease rent is NOT regulated
+        $this->addSql('UPDATE rent_board_case SET rent_board_case.lease_regulated_rent = IF(rent_board_case.lease_regulated_rent IS NULL, 0, 1)');
```
Comment (rimi-itk) [inline migrations/Version20220506111917.php:24]: 👍

File: templates/case/_case_show/_resident_and_rent_board_shareds.html.twig [hunk 2]
```diff
@@ -154,7 +154,14 @@
...
                         <div class="form-group col-md-3">
                             <label>{% trans %}Regulated rent{% endtrans %}</label>
-                            <input type="text" class="form-control" value="{{ case.leaseRegulatedRent }}" readonly>
+                            <input type="text" class="form-control" value="
+                                {% if case.leaseRegulatedRent is not null %}
+                                    {% if case.leaseRegulatedRent %}
+                                        {% trans %}Yes{% endtrans %}
+                                    {% else %}
+                                        {% trans %}No{% endtrans %}
+                                    {% endif %}
```
Comment (rimi-itk) [inline templates/case/_case_show/_resident_and_rent_board_shareds.html.twig:163]: Can be written as `{{ case.leaseRegulatedRent ? 'Yes'|trans : 'No'|trans }}`.

**Review by rimi-itk — APPROVED** [review]
Approved with a single comment.

File: templates/case/_case_show/_resident_and_rent_board_shareds.html.twig (hunk 2 above)
Comment (jekuaitk) [reply templates/case/_case_show/_resident_and_rent_board_shareds.html.twig:163]: The idea was to not display anything if `case.leaseRegulatedRent` was not set (i.e. `null`) and `Yes`/ `No` if either was explicitly set. But using ternary operator does still make sense.

Switched to
`{{ case.leaseRegulatedRent is not null ? (case.leaseRegulatedRent ? 'Yes'|trans : 'No'|trans )}}`

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**